test/nermesterts/game_picker_test.exs

```diff
@@ -0,0 +1,15 @@
+defmodule GamePickerTest do
+  use ExUnit.Case
+
+  alias Nermesterts.Game
+  alias GamePicker
+
+  test "game selection is consistent when the day and number of players are the same" do
+    games = [%Game{name: "Game1", min_players: 2, max_players: 4},
+             %Game{name: "Game2", min_players: 2, max_players: 4},
+             %Game{name: "Game3", min_players: 2, max_players: 4}]
+    randomizer = GamePicker.Randomizer.new({2016, 10, 17})
```

Comment:
This won't work because `new` returns the result of the last called function. Not the `Randomizer`.